webrtc-stats.html

```diff
@@ -3622,14 +3624,16 @@ <h2>
                   measurement does not count the size of the IP or other transport layers like TCP
                   or UDP. It is similar to the TIAS defined in [[!RFC3890]], i.e., it is measured
                   in bits per second and the bitrate is calculated over a 1 second window.
+
+                  For candidate pairs in use, the estimate is normally no lower than the
```

Comment:
Is this paragraph requiring that the 1s interval be calculated as a sliding window of 1s that ends at the lastPacketTimestamp?

We should further clarify this aspect